packages/ethereum-erc20-provider/lib/EthereumErc20Provider.js

```diff
@@ -4,8 +4,8 @@ import { BigNumber } from 'bignumber.js'
 import Provider from '@liquality/provider'
 import { padHexStart } from '@liquality/crypto'
 import {
-  ensureAddressStandardFormat,
-  ensureHexEthFormat
+  toLowerCaseWithout0x,
+  ensure0x
```

Comment:
Not that eventually this might not even be just 0x, but also [EIP-55](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md)